src/Command/CompletionCommand.php

```diff
@@ -97,21 +97,13 @@ public function buildOptionParser(ConsoleOptionParser $parser): ConsoleOptionPar
      */
     public function execute(Arguments $args, ConsoleIo $io): ?int
     {
-        $mode = $args->getArgument('mode');
-        switch ($mode) {
-            case 'commands':
-                return $this->getCommands($args, $io);
-            case 'subcommands':
-                return $this->getSubcommands($args, $io);
-            case 'options':
-                return $this->getOptions($args, $io);
-            case 'fuzzy':
-                return static::CODE_SUCCESS;
-            default:
-                $io->err('Invalid mode chosen.');
-        }
-
-        return static::CODE_SUCCESS;
+        return match ($args->getArgument('mode')) {
+            'commands' => $this->getCommands($args, $io),
+            'subcommands' => $this->getSubcommands($args, $io),
+            'options' => $this->getOptions($args, $io),
+            'fuzzy' => static::CODE_SUCCESS,
+            default => $io->err('Invalid mode chosen.')
```

Comment:
looks this is invalid
previous default was `return static::CODE_SUCCESS;`
but now is result  of `$io->err()`

Reply:
I incorrectly thought that err() returns void, will fix it.